config/development.json

```diff
@@ -52,6 +52,7 @@
 		"domains/cctlds": true,
 		"domains/cctlds/ca": true,
 		"domains/cctlds/fr": true,
+		"domains/cctlds/uk": true,
```

Comment:
As we discussed, I think we should enable this in production as well and rely on the backend to do the release because we can only do harm by having it unavailable: This code does nothing unless someone gets a `.uk` domain into their cart before we expect them to, and if that happens we'll only make the situation worse if we deny them the extra information form.